src/SFA.DAS.EmployerCommitmentsV2.Web/Mappers/Apprentice/ReviewApprenticeshipUpdatesRequestToViewModelMapper.cs

```diff
@@ -49,6 +49,7 @@ public async Task<ReviewApprenticeshipUpdatesViewModel> Map(ReviewApprenticeship
                     {
                         FirstName = update.FirstName,
                         LastName = update.LastName,
+                        //Email = update.E Chas!
```

Comment:
Something looks a little wrong here

Reply:
Hi this was resolved in the next commit.. (needed the Email property adding to the api object GetApprenticeshipUpdatesResponse) https://github.com/SkillsFundingAgency/das-employercommitments-v2/pull/229/commits/20ad73c340c94557c3b01bd61051ff833b07bca7